dartagnan/src/main/java/com/dat3m/dartagnan/wmm/analysis/WmmAnalysis.java

```diff
@@ -28,7 +28,7 @@ public class WmmAnalysis {
             name= LOCALLY_CONSISTENT,
             description="Assumes local consistency for all created wmms.",
             secure=true)
-    private boolean assumeLocalConsistency = true;
+    private boolean assumeLocalConsistency = false;
```

Comment:
Add a comment explaining that local consistency do not hold in PTX

Reply:
Added